snakeplane/helper_classes.py

```diff
@@ -262,6 +263,9 @@ def create_record_info(self) -> sdk.RecordInfo:
         return sdk.RecordInfo(self._engine_vars.alteryx_engine)
 
 
+ChunkStage = Enum("ChunkStage", "none first middle last")
```

Comment:
While I like the brevity of the Enum functional API, I don't like the enumerations being all crammed into a single string.  It lacks the explicitness of the normal object based API.